src/calculator.js

```diff
@@ -1,47 +1,33 @@
-exports._check = () => {
+exports._check = (x,y) => {
```

Comment:
A space is required after &#39;,&#39;.

###### See http://eslint.org/docs/rules/comma-spacing for details about this rule